### PR TITLE
Kan 11 login page forgot password

### PR DESCRIPTION
**Pull request by mailong395**
- Add UI forget password
- Add UI verify email
- config some rule eslint
- clear source
- config path import

### CODE REVIEW

File: public/assets/images/Login/img-login.png
Comment (MaiNhatHoangY2001) [inline public/assets/images/Login/img-login.png:1]: please upload image to ES2

File: src/components/ui/button/button-go-back.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import Button from "@mui/material/Button";
+
+const ButtonGoBack = () => {
+  const navigate = useNavigate();
+
+  return (
+    <Button
+      startIcon={<ArrowBackIos />}
+      variant="text"
+      onClick={() => navigate(-1)}>
```
Comment (MaiNhatHoangY2001) [inline src/components/ui/button/button-go-back.tsx]: What is "-1" in this code ?

File: src/pages/auth/forgot-password/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import ButtonGoBack from "@components/ui/button/button-go-back.tsx";
+import FormForgotPassword from "@pages/auth/forgot-password/form-forgot-password.tsx";
+import {HEADER_FORGOT_PASSWORD} from "@utils/constants.ts";
+
+const ForgotPasswordPage = () => {
+  const {title, subTitle} = HEADER_FORGOT_PASSWORD;
+
+  return (
+    <main className="flex flex-col items-start bg-black/5 p-8 rounded-3xl gap-8 max-w-xl">
+      <ButtonGoBack />
```
Comment (MaiNhatHoangY2001) [inline src/pages/auth/forgot-password/index.tsx]: BackButton ?

File: src/pages/auth/forgot-password/index.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      <AuthHeader title={title} subTitle={subTitle} />
+      <FormForgotPassword />
```
Comment (MaiNhatHoangY2001) [inline src/pages/auth/forgot-password/index.tsx]: ForgotPasswordForm ?

File: src/pages/auth/reset-password/index.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import FormResetPassword from "@pages/auth/reset-password/form-reset-password.tsx";
+import {HEADER_RESET_PASSWORD} from "@utils/constants.ts";
+
+const ResetPasswordPage = () => {
+  const {title, subTitle} = HEADER_RESET_PASSWORD;
+
+  return (
+    <main className="bg-black/5 p-8 rounded-3xl flex flex-col gap-8 max-w-xl">
+      <AuthHeader title={title} subTitle={subTitle} />
+      <FormResetPassword />
```
Comment (MaiNhatHoangY2001) [inline src/pages/auth/reset-password/index.tsx]: ResetPasswordForm ?

File: src/pages/auth/verify-email/form-verify-email.tsx [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+import {authResendOTP, authVerifyEmail} from "@services/api/auth-api.ts";
+import {numberToTime} from "@utils/libs.ts";
+import {VerifyEmailSchema, VerifyEmailType} from "@utils/schema.ts";
+import {isAxiosError} from "axios";
+import {MuiOtpInput} from "mui-one-time-password-input";
+
+const FormVerifyEmail = () => {
+  const navigate = useNavigate();
+  const {email} = useParams();
+  const {current, reset} = useCountdown(0, 60 * 5);
```
Comment (MaiNhatHoangY2001) [inline src/pages/auth/verify-email/form-verify-email.tsx]: define constant for this code like 
```suggestion
  const {current, reset} = useCountdown(0, TIME_END);
```

File: src/pages/auth/verify-email/index.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+import FormVerifyEmail from "@pages/auth/verify-email/form-verify-email.tsx";
+import {HEADER_VERIFY_EMAIL} from "@utils/constants.ts";
+
+const VerifyEmail = () => {
+  const {title, subTitle} = HEADER_VERIFY_EMAIL;
+  return (
+    <main className="max-w-md space-y-8 bg-black/5 p-8 rounded-3xl">
+      <ButtonGoBack />
+      <AuthHeader title={title} subTitle={subTitle} />
+      <FormVerifyEmail />
```
Comment (MaiNhatHoangY2001) [inline src/pages/auth/verify-email/index.tsx]: VerifyEmailForm ?

File: src/utils/libs.ts [hunk 7]
```diff
@@ -0,0 +1,10 @@
+export function numberToTime(seconds: number): string {
```
Comment (MaiNhatHoangY2001) [inline src/utils/libs.ts]: convertNum2Time

**Review by MaiNhatHoangY2001 — CHANGES_REQUESTED** [review]

**Review by MaiNhatHoangY2001 — APPROVED** [review]

**Review by Tuan1O2 — APPROVED** [review]

**Pull request merged**